ops/ansible/playbooks-ccs/build_bfd-server-load.yml

```
@@ -0,0 +1,46 @@
+---
+##
+# Configures the BFD DB Migrator Service for CCS env
+##
+- name: Configure server-load Controller
+  hosts: all
+  become: true
+  remote_user: ec2-user
+  gather_facts: no
+  vars:
+    server_load_dir: /opt/server-load
+    # TODO: Consider removing `env` when SSH/sudoer users have been resolved in SSM BFD-2068
+    env: "test"
+
+  tasks:
+    - name: Download BFD Repository
+      git:
+        repo: https://github.com/CMSgov/beneficiary-fhir-data.git
+        dest: /beneficiary-fhir-data
+        # use hash from `git_commit` when on stable master branch, otherwise use git_branch name
+        version: "{{ git_branch if git_branch != 'master' else git_commit }}"
```

I don't like that this is necessary, but I must have iterated on this a bit to make it more palatable for myself. I'm not sure if anyone else has an opinion, but I suppose my preference is for _x if y is equal_ rather than _x if x is not equal_. What do you think?
```suggestion
        version: "{{ git_commit if git_branch == 'master' else git_branch }}"
```

---

I don't feel _too_ strongly either way -- I think if this is the way the other Ansible `build*` playbooks are doing this check then this playbook should also follow that convention. I think, at-a-glance, `x if y == <something>` is more readable than `x if x != <something>` in this instance.